### DISCUSSION TITLE
Next release plans/date

### DISCUSSION THREAD

**Discussion by keithks** [confluentinc/confluent-kafka-python#1499] [General]
Hi,

Just wanted to ask about plans for the next release?  

We are waiting for the some fixes in master and have been working around them in our python code.  We might build a private off master if the release is still some time off and that's a safe thing to do ourselves

Thanks!

**Comment by ChillarAnand** [comment]
Hi,

I guess next release will have linux-aarch64 builds. Just checking when the new release will be?

**Reply by ChillarAnand** [reply]
@edenhill Any roadmap/schedule that is available publicly for the next release?

**Reply by ChillarAnand** [reply]
https://github.com/confluentinc/confluent-kafka-python/issues/1466#issuecomment-1463521315

Based on this, the new release should be available by early April.

**Reply by ChillarAnand** [reply]
Finally, a new release. Thanks to maintainers.
